mimesniff/mime-type-sniffing/scriptable-mime-types.window.js

```diff
@@ -0,0 +1,57 @@
+// META: timeout=long
+// META: script=/common/subset-tests.js
+// META: variant=?1-738
+// META: variant=?739-1476
+// META: variant=?1477-2214
+// META: variant=?2215-2952
+// META: variant=?2953-3690
+// META: variant=?3691-last
+
+setup({
+  "explicit_timeout": true,
+});
+
+promise_test(() => {
+  return fetch("resources/scriptable-mime-types.json").then(res => res.json().then(runTests));
+}, "Loading data…");
+
+function runTests(tests) {
+  tests.forEach((test_data, test_id) => {
+    subsetTest(async_test, (t) => {
+        const iframe = document.createElement("iframe");
```

Comment:
Oops, that's an artifact of all the trial and error I had to do to avoid prematurely timing out. Forgot to dedent after removing a wrapping block. Good catch.